test/passing/tests/labelled_args-414.ml.ref

```diff
@@ -13,7 +13,7 @@ let () =
 let () =
   very_long_function_name
     ~very_long_argument_label:(* foo *)
-    (fun
+      (fun
```

Comment:
This isn't a regression, this is consistent with fun args that are not the last.